docs/preview/02-Features/01-General/receive-cloudevents-job.md

```diff
@@ -39,6 +39,16 @@ public class Startup
         services.AddCloudEventBackgroundJob(
             subscriptionNamePrefix: "Sub-",
             serviceBusTopicConnectionStringSecretKey: "<secret-key-name-for-servicebus-topic-connection-string>");
+
+        // Add CloudEvent background job via Managed Identity
+        services.AddCloudEventBackgroundJobUsingManagedIdentity(
```

Comment:
Great job in adding an extension fully consistent with the naming conventions we had in other libraries and documentation guidelines. Great, indeed!